src/chapters/02/main.tex

```diff
@@ -489,10 +507,12 @@ \section{Solving with R}\label{sec:solving-with-R}
 discussed above. To do this, we need to use an R library call
 \mintinline{R}{expm}.
 
-To be able to make use of the nice \mintinline{R}{%>%} "pipe" operator we are
+To be able to make use of the nice \mintinline{R}{%>%} ``pipe'' operator we are
 also going to load the \mintinline{R}{magrittr} library.
 Now if we wanted to see what would happen after .5 time units we obtain:
 
+%%% Very very minor point. But the %>% is not required right? I'm just wondering if this is too many new throwaway concepts for one chapter? It is nice though.
```

Comment:
No *but* my understanding of "modern" R is that it essentially is so we should keep it.